pulsar-client-cpp/python/pulsar/functions/serde.py

```diff
@@ -68,9 +68,9 @@ def deserialize(self, input_bytes):
       return pickle.loads(input_bytes)
 
 class IdentitySerDe(SerDe):
-  """Pickle based serializer"""
+  """utf-8 based identity serializer"""
```

Comment:
I don't fully understand here. "identity" typically means not changing things, but the code here is converting between bytes and string, which doesn't seem to be correct to me.

Reply:
I think IdentitySerde is a bad name(probably should have been called simple serde).
I didn't rename it to preserve backwards compatibility(not sure whether its worth preserving)

Reply:
Lets step back to what we are trying to accomplish. The whole idea is to have a default serializer so users dont have to specify their own serializer. Should we rename IdentitySerde to StringSerDe and make that the default?